examples/confluent_cloud.py

```diff
@@ -100,12 +100,12 @@ def acked(err, msg):
             # Initial message consumption may take up to `session.timeout.ms` for
             #   the group to rebalance and start consuming
             continue
-        elif not msg.error():
-            print('consumed: {0}'.format(msg.value()))
-        elif msg.error().code() == KafkaError._PARTITION_EOF:
-            print('end of partition: {0} [{1}] @ {2}'.format(msg.topic(), msg.partition(), msg.offset()))
-        else:
-            print('error: {0}'.format(msg.error().str()))
+        if msg.error():
+            # Most error message are self-recting; log error and continue
```

Comment:
recting?

Reply:
rectifying, sorry. I'm not sure that is the right word either. Just going back to 'typically temporary'